api/envoy/config/overload/v3/overload.proto

```diff
@@ -50,10 +51,18 @@ message ThresholdTrigger {
       "envoy.config.overload.v2alpha.ThresholdTrigger";
 
   // If the resource pressure is greater than or equal to this value, the trigger
-  // will fire.
+  // will enter saturation.
   double value = 1 [(validate.rules).double = {lte: 1.0 gte: 0.0}];
 }
 
+message RangeTrigger {
+  // If the resource pressure is greater than this value, the trigger will enter the scaling state.
```

Comment:
This is the first time the reader encounters "scaling state". It should either be explain in detail or a link to the architecture docs added with `:ref:` added.